test/mongodb.test.js

```diff
@@ -2013,6 +2013,21 @@ describe('mongodb connector', function() {
     });
   });
 
+  it('should export the MongoDB function', function() {
+    var module = require('../');
+    module.MongoDB.should.be.an.instanceOf(Function);
+  });
+
+  it('should export the ObjectID function', function() {
+    var module = require('../');
+    module.ObjectID.should.be.an.instanceOf(Function);
+  });
+
+  it('should export the generateMongoDBURL function', function() {
+    var module = require('../');
+    module.generateMongoDBURL.should.be.an.instanceOf(Function);
+  });
+
```

Comment:
@phairow Could you instead add some tests related to this patch? For example, tests related to `lazyConnect`